test/__mocks__/request-promise.js

```diff
@@ -0,0 +1,13 @@
+const { photo, album, albumList } = require('../fixture/albumPhotos');
+
+module.exports = jest.fn(param => {
+  const splitUri = param.uri.split('?')[1];
+  const photos = param.uri.search('photos');
+  if (photos === -1) {
+    if (splitUri !== undefined) {
```

Comment:
lets add names to this conditions because its not clear why we are returning something different in each case

Reply:
ok!